content/Software Support/IDE Settings/Change-the-language-in-Arduino-IDE.md

```diff
@@ -51,29 +47,14 @@ Alternatively, you can edit the `preferences.txt` file directly.
 
 ---
 
-## Troubleshooting
-
-### If only English is available in IDE 2
-
-Community translations were added to Arduino IDE 2 in version **2.0.0-rc6**.
-
-You can find the latest version on the [Software page](https://www.arduino.cc/en/software).
-
-### Some text is not translated
+## If the language isn't changing
 
-Arduino IDE translations have been made through community contributions, and not all of them are complete. You can see the status of each language at **Transifex**:
-
-* [Arduino IDE 1 translation project](https://www.transifex.com/mbanzi/arduino-ide-15/)
-* [Arduino IDE 2 translation project](https://www.transifex.com/arduino-1/ide2/)
-
```

Comment:
What is the rationale of removing this information?

Reply:
I have added it back in [5b691e3](https://github.com/arduino/help-center-content/pull/276/commits/5b691e3506189b8ef92d6197e99548db4f19b87c) since some languages are not completely translated.